### PR TITLE
Extend Distributed Tracing doc to instruct how to enable the jaeger links

### PR DESCRIPTION
**Pull request by alpeb**
Fixes linkerd/linkerd2#6470

### CODE REVIEW

File: linkerd.io/content/2.10/tasks/distributed-tracing.md [hunk 1]
```diff
@@ -123,13 +123,24 @@ goes between two meshed pods, there will be a total of 4 spans. Two will be on
...
+
+After having set up the Linkerd-Jaeger extension, as the proxy adds application
+meta-data as trace attributes, users can directly jump into related resources
+traces directly from the linkerd-web dashboard by clicking the Jaeger icon in
+the Metrics Table, as shown below:
 
 {{< fig src="/images/tracing/linkerd-jaeger-ui.png"
         title="Linkerd-Jaeger" >}}
 
+To obtain that functionality you need to install (or upgrade) the Linkerd-Viz
```
Comment (cpretzer) [inline linkerd.io/content/2.10/tasks/distributed-tracing.md:136]: nit: should this be `linkerd-viz`?

I don't know if we have a "right" way of referencing the extensions

**Review by cpretzer — APPROVED** [review]
LGTM @alpeb 

Just left a note about the way linkerd-viz may or may not be capitalized

File: linkerd.io/content/2.10/tasks/distributed-tracing.md (hunk 1 above)
Comment (alpeb) [reply linkerd.io/content/2.10/tasks/distributed-tracing.md:136]: I was following the the same spelling used elsewhere in this same doc. I don't have a particular preference, as long as it remains consistent :man_shrugging:

Comment (cpretzer) [reply linkerd.io/content/2.10/tasks/distributed-tracing.md:136]: 👍

**Pull request merged**